database/definition-variablen.txt

```diff
@@ -0,0 +1,84 @@
+Variable
+
+# Approved
+* "yes": this shall be put online, it is approved
+* "no": this one is confirmed to happen but misses some important information
+* "nada": this one is not going to happen this year
+* "": keep on checking this if there are news
+
+# Datestart: YYYYMMDD
+# Datenend: YYYYMMDD
+
+# Label: the name we use in the list
+# Name: The name we use in the detailed view
+# Shortname: no use yet, usable in social media etc
+# Hashtag: to be used in social media
+# Homepage: URL
+# City: English City name (could be changed one day to an international id, let's see ...)
+# Country: the two-digit country code
+# Venue: Name of the venue
```

Comment:
Just the name or also address? E.g. just "Lyon Convention Center" or "50 Quai Charles de Gaulle" added?

Reply:
see db, only name, let's not overcomplicate